--- origsrc/newlib-2.2.0/libgloss/lm32/Makefile.in	2008-12-17 17:08:09.000000000 -0600
+++ src/newlib-2.2.0/libgloss/lm32/Makefile.in	2014-12-25 18:15:32.772093800 -0600
@@ -72,10 +72,10 @@ distclean maintainer-clean realclean: cl
 .PHONY: install info install-info clean-info
 install: install-scripts # install-includes 
 	@for crt0 in ${CRT0S}; do\
-	 $(INSTALL_PROGRAM) $${crt0} $(tooldir)/lib${MULTISUBDIR}; \
+	 $(INSTALL_PROGRAM) $${crt0} $(DESTDIR)$(tooldir)/lib${MULTISUBDIR}; \
 	done
 	@for bsp in ${BSPS}; do\
-	 $(INSTALL_PROGRAM) $${bsp} $(tooldir)/lib${MULTISUBDIR}; \
+	 $(INSTALL_PROGRAM) $${bsp} $(DESTDIR)$(tooldir)/lib${MULTISUBDIR}; \
 	done
 
 install-includes:
@@ -86,7 +86,7 @@ install-includes:
 	
 install-scripts:
 	@for script in ${SCRIPTS}; do\
-	 $(INSTALL_DATA) ${srcdir}/$${script} $(tooldir)/lib${MULTISUBDIR}; \
+	 $(INSTALL_DATA) ${srcdir}/$${script} $(DESTDIR)$(tooldir)/lib${MULTISUBDIR}; \
 	done
 	
 doc:
--- origsrc/newlib-2.2.0/libgloss/mep/Makefile.in	2009-04-21 13:13:32.000000000 -0500
+++ src/newlib-2.2.0/libgloss/mep/Makefile.in	2014-12-25 18:33:42.316948500 -0600
@@ -99,13 +99,13 @@ crtn.o: $(srcdir)/crtn.S $(srcdir)/sim-c
 
 install: $(CRT) $(SIM_BSP)
 	for c in $(CRT) $(SIM_BSP); do \
-	  $(INSTALL_DATA) $$c $(tooldir)/lib${MULTISUBDIR}/$$c ;\
+	  $(INSTALL_DATA) $$c $(DESTDIR)$(tooldir)/lib${MULTISUBDIR}/$$c ;\
 	done
 	if test -z "$(MULTISUBDIR)"; then \
 	  for c in $(SCRIPTS); do \
-	    $(INSTALL_DATA) $(srcdir)/$$c $(tooldir)/lib${MULTISUBDIR}/$$c ;\
+	    $(INSTALL_DATA) $(srcdir)/$$c $(DESTDIR)$(tooldir)/lib${MULTISUBDIR}/$$c ;\
 	  done; \
-	  $(INSTALL_DATA) $(srcdir)/gmap_default.ld $(tooldir)/lib/gmap_default.tmpl; \
+	  $(INSTALL_DATA) $(srcdir)/gmap_default.ld $(DESTDIR)$(tooldir)/lib/gmap_default.tmpl; \
 	else true; fi
 
 clean mostlyclean:
--- origsrc/newlib-2.2.0/libgloss/mt/Makefile.in	2006-01-16 17:23:30.000000000 -0600
+++ src/newlib-2.2.0/libgloss/mt/Makefile.in	2014-12-25 18:33:59.865176800 -0600
@@ -103,11 +103,11 @@ trap.o:	$(srcdir)/trap.S
 
 install: $($(CPU)_INSTALL)
 	for c in $(CRT0); do \
-	  $(INSTALL_DATA) $$c $(tooldir)/lib${MULTISUBDIR}/$$c ; \
+	  $(INSTALL_DATA) $$c $(DESTDIR)$(tooldir)/lib${MULTISUBDIR}/$$c ; \
 	done; 
-	$(INSTALL_DATA) $(SIM_BSP) $(tooldir)/lib${MULTISUBDIR}/$(SIM_BSP)
+	$(INSTALL_DATA) $(SIM_BSP) $(DESTDIR)$(tooldir)/lib${MULTISUBDIR}/$(SIM_BSP)
 	for c in $(SCRIPTS); do \
-	  $(INSTALL_DATA) $(srcdir)/$$c $(tooldir)/lib/$$c ; \
+	  $(INSTALL_DATA) $(srcdir)/$$c $(DESTDIR)$(tooldir)/lib/$$c ; \
 	done;
 
 clean mostlyclean:
--- origsrc/newlib-2.2.0/libgloss/rl78/Makefile.in	2011-11-29 00:33:48.000000000 -0600
+++ src/newlib-2.2.0/libgloss/rl78/Makefile.in	2014-12-25 18:32:44.173065100 -0600
@@ -135,11 +135,11 @@ write.o : $(SDEPS)
 
 install: $(CRT) $(SIM_BSP) $(SCRIPTS)
 	for c in $(CRT) $(SIM_BSP); do \
-	  $(INSTALL_DATA) $$c $(tooldir)/lib${MULTISUBDIR}/$$c ;\
+	  $(INSTALL_DATA) $$c $(DESTDIR)$(tooldir)/lib${MULTISUBDIR}/$$c ;\
 	done
 	for c in $(SCRIPTS); do \
 	  b=`basename $$c`; \
-	  $(INSTALL_DATA) $$c $(tooldir)/lib${MULTISUBDIR}/$$b ;\
+	  $(INSTALL_DATA) $$c $(DESTDIR)$(tooldir)/lib${MULTISUBDIR}/$$b ;\
 	done
 
 clean mostlyclean:
